priv/cabbage/apps/itest/lib/reorg.ex

```diff
@@ -0,0 +1,101 @@
+defmodule Itest.Reorg do
```

Comment:
Regarding codebase style:
I think our convention is that we put genserver API, like `start_reorg/0` at the top of the module (so that it's clearly evident how one can interact with a process).
Also, not sure why this didn't trigger credo... parentheses (even for functions with 0 arity) are mandatory.

Feature request:
Are you able to detect a reorg and it's depth? perhaps using: https://docs.docker.com/engine/api/sdk/examples/#print-the-logs-of-a-specific-container

This would help observability of tests greatly!

Your question regarding where do we trigger reorgs:
I think every test needs to survive reorgs! The question is, how do we enable/disable these. Maybe with an environment variable?
I think it's important to be able to do that - because you don't want to even start `Itest.Reorg` when you're running test towards one node geth right? And I don't imagine devs running reorg tests locally often.

Reply:
I addressed your comments. Logs are added as a separate background process